cmd/conformance/posix/main.go

```diff
@@ -58,11 +59,11 @@ func main() {
 	s, a := getSignersOrDie()
 
 	// Create the Tessera POSIX storage, using the directory from the --storage_dir flag
-	storage, err := posix.New(ctx, *storageDir, *initialise, tessera.WithCheckpointSigner(s, a...), tessera.WithBatching(256, time.Second))
+	storage, err := posix.New(ctx, *storageDir, *initialise, tessera.WithCheckpointSigner(s, a...), tessera.WithBatching(layout.TileWidth, time.Second))
```

Comment:
I wouldn't replace this one; 256 here is kinda arbitrary, and I'd be worried about accidentally creating some "guideline" about how this number must be chosen.

Reply:
Done.